.travis.yml

```diff
@@ -1,16 +1,13 @@
 language: php
-
 sudo: false
-
 cache:
     directories:
         - $HOME/.composer/cache/files
         - $HOME/symfony-bridge/.phpunit
 
 env:
     global:
-        - TEST_COMMAND="composer test"
```

Comment:
The php-http organisation has (maybe silently) decided that all our packages should be tested with `composer test` and `composer test-ci`. 

We do not need the `PHPUNIT_FLAGS`. You should revert this and continue using `TEST_COMMAND`

Reply:
Ok. I pushed a better version IMHO. Let's see what's the build result.